code/modules/mob/living/carbon/emote.dm

```diff
@@ -134,6 +134,23 @@
 		qdel(N)
 		to_chat(user, span_warning("You're incapable of slapping in your current state."))
 
+/datum/emote/living/carbon/steal
+	key = "steal"
```

Comment:
Should go instead with `shoesteal` (this was on my TODO list) so that it's obvious what it's for. `*steal` is really generic and confusing.

Reply:
i was leaving it generic in case more should be added but this is fine too